docs/source/tutorials/extending_mantid_with_python/advanced_fit_behaviours/05_exercise_6.rst

```diff
@@ -0,0 +1,77 @@
+.. _05_exercise_6:
+
+==========
+Exercise 6
+==========
+
+The aim of this exercise is to implement a function to fit the output data
+from :ref:`05_exercise_4`. For simplicity a solution file, *11001_deltaE.nxs*,
+is provided with the training data.
+
+The peak can be fairly well approximated using a Lorentz function:
+
+.. math::
+
+    \frac{A}{π}(\frac{\frac{Γ}{2}}{(x-c)^2 + (\frac{Γ}{2})^2})
+
+where ``A`` is the amplitude, ``Γ`` is the full width at half maximum and
+``c`` is the peak centre. We will first define this as a simple 1D function
+and then improve it to use the peak function capabilities.
+
+Simple 1D
+=========
+
+* Define a new 1D function called ``Lorentz``.
+* It should have 3 parameters corresponding to the parameters described above.
+* Write the ``function1D`` method that evaluates the required values from the
+  input x data using the definition as above. (Hint: you can use the python
+  math module for
+  `pi <https://docs.python.org/2/library/math.html#constants>`_).
+
+Test this implementation:
+
+#. Load the data file.
+#. Plot the spectrum.
+#. Use the fit browser (using the fit function tool button (looks like a peak
+   with a vertical red line on top)).
+#. Right click on plot and select "Add other function...".
+
+* You'll want to see how the fit progresses so set the log level in the
+  results window (View->Results Log) to information by right clicking in the
```

Comment:
I'd call it the Messages Box instead of the results window (a change from MantidPlot to Workbench)
Feel free to change the View->Results to Messages too, or just remove this bit in brackets!